wg-qa/README.md

```diff
@@ -0,0 +1,34 @@
+# Releases WG
+
+Oversees stability, quality, and testing criteria within Electron.
+
+## Membership
+
+| Name |
+| -----------------|
+| Maria (Pending)  |
+| Keeley (Pending) |
+| David            |
+| Alexey           |
+| Sofia            |
+
+## Chair
```

Comment:
For this section, see [Releases WG readme](https://github.com/electron/governance/blob/master/wg-releases/README.md) for an example on how to format the Membership section. We would like all the WGs to have consistent READMEs 🙂

Reply:
Fixed